--- /Amethyst-iOS/Natives/MinecraftResourceDownloadTask.m
+++ /Amethyst-iOS/Natives/MinecraftResourceDownloadTask.m
@@ -315,14 +315,15 @@
 // Check if the account has permission to download
 - (BOOL)checkAccessWithDialog:(BOOL)show {
     // for now
-    BOOL accessible = [BaseAuthenticator.current.authData[@"username"] hasPrefix:@"Demo."] || BaseAuthenticator.current.authData[@"xboxGamertag"] != nil;
-    if (!accessible) {
-        [self.progress cancel];
-        if (show) {
-            [self finishDownloadWithErrorString:@"Minecraft can't be legally installed when logged in with a local account. Please switch to an online account to continue."];
-        }
-    }
-    return accessible;
+    // BOOL accessible = [BaseAuthenticator.current.authData[@"username"] hasPrefix:@"Demo."] || BaseAuthenticator.current.authData[@"xboxGamertag"] != nil;
+    // if (!accessible) {
+    //     [self.progress cancel];
+    //     if (show) {
+    //         [self finishDownloadWithErrorString:@"Minecraft can't be legally installed when logged in with a local account. Please switch to an online account to continue."];
+    //     }
+    // }
+    // return accessible;
+    return true;
 }
 
 // Check SHA of the file