--- /Amethyst-Android/app_pojavlauncher/src/main/java/net/kdt/pojavlaunch/Tools.java
+++ /Amethyst-Android/app_pojavlauncher/src/main/java/net/kdt/pojavlaunch/Tools.java
@@ -1703,12 +1703,13 @@ public final class Tools {
         return currentProfile == null || currentProfile.isLocal();
     }
     public static boolean hasOnlineProfile(){
-        for (MinecraftAccount accountToCheck : getAllProfiles()) {
-            if (!accountToCheck.isLocal() && !accountToCheck.isDemo()) {
-                return true;
-            }
-        }
-        return false;
+        // for (MinecraftAccount accountToCheck : getAllProfiles()) {
+        //     if (!accountToCheck.isLocal() && !accountToCheck.isDemo()) {
+        //         return true;
+        //     }
+        // }
+        // return false;
+        return true;
     }